.github/workflows/_rust.yml

```diff
@@ -106,10 +106,12 @@ jobs:
         run: pnpm tailwindcss -i src/input.css -o src/output.css
       - name: Build client
         run: cargo build -p firezone-gui-client
-      - name: Run smoke test
-        run: cargo run -p firezone-gui-client -- smoke-test
-      - name: Test that the crash handler produces a crash dump
-        run: bash scripts/crash-handling-smoke-test.bash
+      - name: Run smoke tests (Linux)
+        if: ${{ runner.os == 'Linux' }}
+        run: bash ../../scripts/tests/smoke-test-gui-linux.sh
```

Comment:
This isn't called yet because the GUI client doesn't compile for Linux yet in this branch.